deployments/deploy-ssh.sh

```diff
@@ -230,7 +242,7 @@ if [ -n "${K8S_VENDOR}" ]; then
         rancher)
             K8S_VENDOR_KEY=node-role.kubernetes.io/controlplane
             K8S_VENDOR_OPERATOR="In"
-            K8S_VENDOR_VALUE='values: ["true"]'
```

Comment:
why we changed rancher? this has nothing to do with EKS

Reply:
by bad, I change it

Reply:
fixed